### PR TITLE
Update changelog for multiport

### PR DESCRIPTION
**Pull request by ndhanushkodi**

### CODE REVIEW

**Review by david-yu — APPROVED** [review]

**Review by thisisnotashwin — APPROVED** [review]
Great job on the PR!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ FEATURES:
     a random UUID: `vault kv put secret/consul/replication key="$(uuidgen)"`. You will need to then provide this secret to both the primary
     and the secondary datacenters with `global.acls.replicationToken` values and allow the `global.secretsBackend.vault.manageSystemACLsRole` Vault role to read it.
     In the primary datacenter, the Helm chart will create the replication token in Consul using the UUID as the secret ID of the token.
+* Multi-port Pods: Support workaround for pods with multiple ports, by registering a Consul service and injecting an Envoy sidecar and init container per port. [[GH-1012](https://github.com/hashicorp/consul-k8s/pull/1012)]
```
Comment (ishustava) [inline CHANGELOG.md:9]: I think we usually use the component as prefix?
```suggestion
* Connect: Support workaround for pods with multiple ports, by registering a Consul service and injecting an Envoy sidecar and init container per port. [[GH-1012](https://github.com/hashicorp/consul-k8s/pull/1012)]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ FEATURES:
...
+  * Transparent proxying, metrics, and metrics merging are not supported for multi-port pods.
+  * Multi-port pods should specify annotations in the format, such that the service names and port names are correspond with each other in the specified order. i.e (`web` service is listening on `8080`, `web-admin` service is listening on `9090`)
```
Comment (ishustava) [inline CHANGELOG.md:11]:
```suggestion
  * Multi-port pods should specify annotations in the format, such that the service names and port names correspond with each other in the specified order, i.e. `web` service is listening on `8080`, `web-admin` service is listening on `9090`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ FEATURES:
...
+    * `consul.hashicorp.com/connect-service': 'web,web-admin`
+    * `consul.hashicorp.com/connect-service': '8080,9090`
```
Comment (ishustava) [inline CHANGELOG.md:13]:
```suggestion
    * `consul.hashicorp.com/connect-service-port': '8080,9090`
```

**Review by ishustava — APPROVED** [review]
Looks great! I've left a couple of minor edits.

**Pull request merged**